packages/run/test/commands/logs.test.ts

```diff
@@ -0,0 +1,10 @@
+import {expect, test} from '@oclif/test'
+
+describe('logs', () => {
+  test
+    .stdout()
+    .command(['logs', '--app=heroku-run-test-app'])
+    .it('shows the logs', ctx => {
+      expect(ctx.stdout).to.match(/^20/)
```

Comment:
What does `/^20/` refer to here?

Reply:
This is a direct port of the original test. It looks like it's asserting that logs are actually printed by checking for the beginning of the year (e.g. `2019-08-19`) in stdout.

Reply:
Ahh gotcha, can we add a comment above or an assertion message? It's a small hint that I think could give a lot of context in the future.

Reply:
Great idea

Reply:
Added a comment explaining this test